app/assets/stylesheets/splash.scss

```diff
@@ -0,0 +1,25 @@
+.splash_container {
+  // background-image: image-url('chalkboard.jpg');
+  // background-size: 100%;
+  // background-repeat: no-repeat;
+  background-color: #000000;
+  background-image: image-url('chalkboard.jpg');
+  background-size: 100%;
```

Comment:
Properties should be ordered background-color, background-image, background-position, background-repeat, background-size, bottom, left, position, right, top, z-index